run.py

```diff
@@ -13,25 +13,14 @@
 from coffea import hist, processor
 from coffea.util import load, save
 
+import modules.topeft as topeft
```

Comment:
Should we consider changing the name of the modules directory to be something less generic, maybe "topcoffea"?

Reply:
I would leave that name "modules" for the folder containing the python modules with multipurpose functions but wouldn't move the topeft.py file there, as it is not a python module with classes of functions that might be used by other scripts, but an AnalysisProcessor to be executed in coffea. So I would keed the "modules" folder as it was and move that script to another less generic folder (as it was, or in another "analyses" folder if you prefer).